packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-commitment.tsx

```diff
@@ -0,0 +1,210 @@
+        },
+      ),
+      [CommitmentStatus.REQUESTED]: translate(
+        this.localeStrings,
+        'manager_billing_subscription_engagement_status_commitement_pending',
+        {
+          nextBillingDate: formatDate(
+            this.serviceDetails?.billing.engagementRequest
+              ?.requestDate as string,
+            this.language,
+          ),
+        },
+      ),
+      [CommitmentStatus.NONE]: '',
+      [CommitmentStatus.ENDS]: '',
+    };
+    return labelByStatus[this.commitmentStatus];
+  }
+
+  render() {
+    const description = this.getDescription();
+    return (
+      <Host>
+        <osds-divider separator />
+        <osds-text
+          class="tile-title"
+          level={OdsThemeTypographyLevel.heading}
+          size={OdsThemeTypographySize._200}
+          color={OdsThemeColorIntent.text}
+        >
+          {this.localeStrings.manager_billing_subscription_engagement}
+        </osds-text>
+        {!this.serviceDetails ? (
+          <osds-skeleton />
+        ) : (
+          <>
+            {description && (
+              <osds-text
+                class="tile-description"
+                level={OdsThemeTypographyLevel.body}
+                size={OdsThemeTypographySize._200}
+                color={OdsThemeColorIntent.default}
+              >
+                {description}
+              </osds-text>
+            )}
+            {this.commitmentStatus === CommitmentStatus.ENDS && (
+              <div>
+                <osds-chip
+                  color={OdsThemeColorIntent.error}
+                  size={OdsChipSize.sm}
+                  variant={OdsChipVariant.flat}
+                >
+                  {translate(
+                    this.localeStrings,
+                    'manager_billing_subscription_engagement_status_engaged',
+                    {
+                      endDate: this.nextBillingDate,
+                    },
+                  )}
+                </osds-chip>
+              </div>
+            )}
+            {this.commitmentStatus === CommitmentStatus.NONE && (
+              <osds-text
+                class="tile-description"
+                level={OdsThemeTypographyLevel.body}
+                size={OdsThemeTypographySize._200}
+                color={OdsThemeColorIntent.default}
+              >
+                <osds-chip
+                  color={OdsThemeColorIntent.error}
+                  size={OdsChipSize.sm}
+                  variant={OdsChipVariant.flat}
+                >
+                  {
+                    this.localeStrings
+                      .manager_billing_subscription_engagement_status_none
+                  }
+                </osds-chip>
+                <osds-link
+                  class="resub-link"
+                  data-tracking={this.commitmentDataTracking}
+                  color={OdsThemeColorIntent.primary}
+                  href={`https://www.ovh.com/manager/dedicated/#/${this.servicePath}/dashboard/commitment`}
```

Comment:
**nitpick:** We cannot be sure that the product will always be part of `dedicated` universe. The URL has to be an input to the `billing-component`.

Reply:
Eventually, we could pass the universe as a parameter to build the url if it changes from product to product. But passing the whole url is not a good idea I think because it makes it more complex to use the component.